SCOPES.md

```diff
@@ -0,0 +1,45 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:beetmover`
+  * Used for Firefox/Fennec/Devedition style beetmover tasks
+
+* `project:comm:thunderbird:releng:beetmover`
+  * Used for Thunderbird style balrog tasks
+
+
+# Scopes
+* `{scope_prefix}:action:push-to-candidates`
+  * Tells beetmoverscript to Schedule a release to ship on balrog channel(s)
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidate-tasks`
+
+* `{scope_prefix}:action:push-to-release`
+  * Tells beetmoverscript to submit locale information for a release.
```

Comment:
Slippery copy-paste from balrogscript maybe? Beetmoverscript `push-to-releases` means we're copying artifacts from candidates directory to releases directory, hence I'd rephrase this to something like: "Tells beetmoverscript to move the artifacts from `candidates` directory to `releases` directory for the corresponding release"